pkgs/data/misc/papirus-folders/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "papirus-folders";
+  version = "1.12.0";
+
+  src = fetchFromGitHub {
+    owner = "PapirusDevelopmentTeam";
+    repo = "papirus-folders";
+    rev = "v${version}";
+    sha256 = "sha256-ZZMEZCWO+qW76eqa+TgxWGVz69VkSCPcttLoCrH7ppY=";
+  };
+
+  makeFlags = [ "PREFIX=${placeholder "out"}" ];
+
+  meta = with lib; {
```

```suggestion
  meta = with lib; {
    description = "Change the color of folders in Papirus icon theme";
```

Short `description`s are useful because some nix search options show them.

---

corrected